grainlearning/rnn/README.md

```diff
@@ -11,7 +11,7 @@ During the installation of grainLearning activate extra `rnn`:
 
 `poetry install --extras "rnn"` or `pip install .[rnn]`
 
-*Note:* This will install a version of tensorflow depending on your system. If gpu dependencies are not installed or activated, it will default to CPU version. Check the installation of tensorflow and/or re-install it following specific instructions for your hardware.
+*Note:* This will install a version of tensorflow depending on your system. If gpu dependencies are not installed or activated, it will default to **CPU** version. Check the installation of tensorflow and/or re-install it following specific instructions for your hardware.
```

Comment:
Add a link on the installation of GPU dependencies?

Reply:
I thought about it but tensorflow changes such instructions often, and they have many different possibilities thus, I think is best just to leave it to the user, otherwise they are going to come to us: the specific set of instructions that you suggested don't work anymore... 
There are enough blogpposts on the internet explaining how to install tensorflow for specific hardware.

Reply:
Replying to the last comment: GPU dependencies are not handled by poetry, only CPU version of tensorflow is installed. That is why such piece of information is on the README.